tuf/api/metadata.py

```diff
@@ -273,6 +276,60 @@ def sign(
 
         return signature
 
+    def verify_delegate(
+        self,
+        role_name: str,
+        delegate: "Metadata",
+        signed_serializer: Optional[SignedSerializer] = None,
+    ):
+        """Verifies that 'delegate' is signed with the required threshold of
+        keys for the delegated role 'role_name'.
+
+        Args:
+            role_name: Name of the delegated role to verify
+            delegate: The Metadata object for the delegated role
```

Comment:
My reading of this is that `verify_delegate` takes an identifier for the delegated role (role_name) and an instance of the delegated role (delegate) – I would expect these two named parameters to have similar names.
* `delegated_role` and `delegated_metadata`?
* `role_name` and `role_metadata`? (handily avoiding the delegate, delegated, delegatee, delegator confusion)